tests/transports-governance/test_governance_usage_tracking_hierarchical.py

```
@@ -0,0 +1,1001 @@
+        # Create VK
+        vk_data = test_data_factory.virtual_key_data()
+        vk_response = governance_client.create_virtual_key(**vk_data)
+        vk_id = vk_response.json()["id"]
+        track_created_entity("virtual_keys", vk_id)
+
+        # Get VK (potentially cached)
+        vk_get1 = governance_client.get_virtual_key(vk_id)
+        initial_data = vk_get1.json()
+
+        # Update VK
+        update_data = {"description": f"Updated at {datetime.now().isoformat()}"}
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)